src/components/atoms/icon/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Icon = ({ className, icon }) => <img className={className} src={icon} alt="close" />
```

Comment:
The alt message should contain a more reliable message to the icon purpose, doesn't?

Reply:
I think it's better to change ``alt`` to be a prop